mcs/class/System.Windows.Forms/System.Windows.Forms/ToolStrip.cs

```diff
@@ -743,6 +742,15 @@ protected override void Dispose (bool disposing)
 						this.overflow_button.drop_down.Dispose ();
 
 					ToolStripManager.RemoveToolStrip (this);
+
+					if (tooltip_timer != null)
+					{
+						tooltip_timer.Stop();
+						tooltip_timer.Dispose();
```

Comment:
Is there any reason why you sometime call just Dispose and other times you call Stop and then Dispose? I would expect the Dispose to be sufficient.

Reply:
That's true. `Dispose` includes `Stop`. In `ToolStrip.cs` I copied the `Stop` from `CloseToolTip` which previously was called. Let me try to update the PR.